src/main/java/com/balako/onlinebookstore/service/BookService.java

```diff
@@ -10,4 +11,10 @@ public interface BookService {
     List<BookDto> findAll();
 
     BookDto getById(Long id);
+
+    void deleteById(Long id);
+
+    BookDto updateBookById(Long id, CreateBookRequestDto requestDto);
```

Comment:
`updateBookById` could be renamed to `update`

Reply:
resolved in dev-6 branch